data/resources/style.css

```diff
@@ -160,29 +160,30 @@ messageindicators {
 }
 
 messagelabel {
-  margin: 6px 12px;
-  font-size: 0.95em;
```

Comment:
IMHO, I would also make the font size of chat titles normal. Then this looks much more harmonic.

Reply:
Yeah, I agree. But I'll do that in a separate PR because I'm also experimenting other style changes in the sidebar.